docs/guidance/map/awards_contracts_buyers_suppliers.md

```diff
@@ -16,12 +16,7 @@ The [UNCITRAL Model Law on Public Procurement (2011)](https://uncitral.un.org/en
 
 In OCDS, the `Award` object is intended to communicate a direct relationship between items, suppliers, and values. It ought to be possible to know, at the award stage, in OCDS data, which items will later be supplied by which suppliers, and what the value of those contracts will be.
 
-```{eval-rst}
-.. admonition:: Note
-   :class: note
-
-   .. markdown::
-
+```{note}
```

Comment:
Is this the same output as:

```
.. admonition:: Note
   :class: note
```

Reply:
That's my understanding, but I can double check if need be. Just seemed like a simplification without changing the output but if does then i can revert